tests/system/python/packages/test_omf_north_service.py

```diff
@@ -380,7 +380,7 @@ def test_omf_service_with_reconfig(self, reset_fledge, start_south_north, read_d
         put_url = "/fledge/category/{}".format(north_service_name)
         resp = utils.put_request(fledge_url, urllib.parse.quote(put_url), data)
         assert "Admin" == resp["PIWebAPIUserId"]["value"]
-
+        time.sleep(5)
```

Comment:
We should add comment here; Why do we need extra delay?

" As with JIRA number; delay in ...."